<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Converter</class>
 <widget class="QWidget" name="Converter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>300</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>ProHex Converter</string>
  </property>
  <widget class="QLabel" name="DecLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>WhatsThisCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string comment="Decimal number">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Decimal value&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string comment="qwe" extracomment="ewq"/>
   </property>
   <property name="accessibleDescription">
    <string comment="asd" extracomment="dsa"/>
   </property>
   <property name="text">
    <string>Dec</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>WhatsThisCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string comment="Hexadecimal number">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hexadecimal value&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>WhatsThisCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string comment="Binary number">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Binary value&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Bin</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DecInput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string comment="reh" extracomment="her"/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>18</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="HexInput">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="BinInput">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>WhatsThisCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string comment="ASCII character">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ASCII Character&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Char</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="LittleEndianGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>691</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Types</string>
   </property>
   <widget class="QLineEdit" name="Int8LEInput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;signed int8&lt;/p&gt;&lt;p&gt;C: char, signed char&lt;/p&gt;&lt;p&gt;C#: sbyte&lt;/p&gt;&lt;p&gt;8 bits (1 byte): -128 ... 127&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>int8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;signed int16&lt;/p&gt;&lt;p&gt;C: short, signed short&lt;/p&gt;&lt;p&gt;C#: short, char&lt;/p&gt;&lt;p&gt;16 bits (2 bytes): -32.768 ... 32.767&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>int16</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;signed int32&lt;/p&gt;&lt;p&gt;C: int, signed int&lt;/p&gt;&lt;p&gt;C#: int&lt;/p&gt;&lt;p&gt;32 bits (4 bytes): -2,147,483,648 ... 2,147,483,647&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>int32</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;signed int64&lt;/p&gt;&lt;p&gt;C: long long, signed long long&lt;/p&gt;&lt;p&gt;C#: long&lt;/p&gt;&lt;p&gt;64 bits (8 bytes): -9,223,372,036,854,775,808 ... 9,223,372,036,854,775,807&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>int64</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;float32&lt;/p&gt;&lt;p&gt;C: float&lt;/p&gt;&lt;p&gt;C#: float&lt;/p&gt;&lt;p&gt;32 bits (4 bytes): 1.2E-38 ... 3.4E+38&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>float32</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Float32LEInput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Int64LEInput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Int32LEInput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Int16LEInput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;unsigned int8&lt;/p&gt;&lt;p&gt;C: char, unsigned char&lt;/p&gt;&lt;p&gt;C#: byte&lt;/p&gt;&lt;p&gt;8 bits (1 byte): 0 ... 255&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>uint8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Uint8LEInput">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Uint16LEInput">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Uint32LEInput">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Uint64LEInput">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Float64LEInput">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;float64&lt;/p&gt;&lt;p&gt;C: double&lt;/p&gt;&lt;p&gt;C#: double&lt;/p&gt;&lt;p&gt;64 bits (8 bytes): 2.3E-308 ... 1.7E+308&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>float64</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;unsigned int16&lt;/p&gt;&lt;p&gt;C: unsigned short&lt;/p&gt;&lt;p&gt;C#: ushort, uchar&lt;/p&gt;&lt;p&gt;16 bits (2 bytes): 0 ... 65535&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>uint16</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;unsigned int32&lt;/p&gt;&lt;p&gt;C: unsigned int&lt;/p&gt;&lt;p&gt;C#: uint&lt;/p&gt;&lt;p&gt;32 bits (4 bytes): 0 ... 4,294,967,295&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>uint32</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;unsigned int64&lt;/p&gt;&lt;p&gt;C: unsigned long long&lt;/p&gt;&lt;p&gt;C#: ulong&lt;/p&gt;&lt;p&gt;64 bits (8 bytes): 0 ... 18,446,744,073,709,551,615&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>uint64</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="CharInput">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="hexCheckbox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>16</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Toggle Hex Mode&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>Toggle all possible fields to HEX Mode</string>
   </property>
   <property name="accessibleName">
    <string>HEX Mode</string>
   </property>
   <property name="accessibleDescription">
    <string>Toggle all possible fields to HEX Mode</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>LittleEndianGroup</zorder>
  <zorder>DecLabel</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>DecInput</zorder>
  <zorder>HexInput</zorder>
  <zorder>BinInput</zorder>
  <zorder>label_7</zorder>
  <zorder>CharInput</zorder>
  <zorder>hexCheckbox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
